opencga-app/app/cloud/docker/opencga-init/override_yaml.py

```diff
@@ -74,6 +75,13 @@
         storage_config["clinical"]["hosts"].clear()
     storage_config["clinical"]["hosts"].insert(i, clinical_host.strip())
 
+# Inject RGA hosts
+rga_hosts = args.rga_hosts.replace('\"','').replace('[','').replace(']','').split(",")
```

Comment:
It'd have been nice to extract this into a method, as this same code is triplicated now.

Reply:
I have removed the code to a method, I have adjusted it to reuse it in 5 places